utils/sl/env_modules.py

```diff
@@ -110,8 +110,8 @@ def _init_module_func(self, default_func):
     def _init_mpi_module(self):
         """init mpi module function"""
         try:
-            subprocess.check_call(['sh', '-l', '-c', 'module -V'],
-                                  stdout=DEVNULL, stderr=DEVNULL)
+            subprocess.check_call(['sh', '-l', '-c', 'module -V'], \
+                                  stdout=DEVNULL, stderr=DEVNULL)  #nosec
```

Comment:
Specific reason we have "#nosec" in the lines above?

Reply:
From testing, full path of /usr/bin/sh causing failure in build:
Test stage Build on Ubuntu 18.04 with Clang completed with status FAILURE. https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-2602/7/execution/node/198/log